packages/riipen-ui/src/components/Input.jsx

```diff
@@ -125,14 +132,36 @@ const Input = ({
           border-color: ${theme.palette.negative.main};
         }
 
-        .underlined {
-          background-color: transparent;
-          border: none;
-          border-bottom: 1px solid rgba(0, 0, 0, 0.23);
+        .icon {
+          color: ${theme.palette.grey[500]};
+          left: ${theme.spacing(4)}px;
+          position: absolute;
```

Comment:
I guess the question is should we allow icons to be any color people want? I think this is fine though. Unless anyone else wants to change this.

Reply:
Yeah, wasn't sure on forcing the colour